app/assets/javascripts/home.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```

Comment:
Best practice would be to delete all these files you're not using. They're automatically created when you do ```rails generate``. I heard this feedback in a code review from C3, and I never followed their advice, but I'm still passing it on so you can decide for yourselves :P